docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2021-01-19
+ - Remove deprecated python 3.5.
```

Comment:
I'd suggest something like: `Python 3.5 is no longer supported.`

Reply:
@thc202 feel free to use github inline suggestions ike the following :)

```suggestion
 - Python 3.5 is no longer supported 3.5.
```

Reply:
@thc202 :heavy_check_mark:

Reply:
Those create unnecessary commits which don't pass the DCO check.

Note that it was added a 3.5 at the end.

Reply:
@thc202 thanks ;) 

> Those create unnecessary commits which don't pass the DCO check

right, I'll look for a solution (maybe squash+merge?)

Reply:
That requires clean up of the commit message.